### PR TITLE
Enable circleci testing

### PR DESCRIPTION
**Pull request by Xorima**
### Description

Enables circleci testing on the redisio cookbook

### Issues Resolved

N/A

### Contribution Checklist

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: Please add labels to this Pull Request
```
 #  from Dangerfile:28
 #  -------------------------------------------
 #  
 >  raise 'Please add labels to this Pull Request' if github.pr_labels.empty?
 #  
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by Xorima** [comment]
@damacus when you get time can you hook up the circleci backend on this one please?

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  end
+  false
+end
+
+raise 'Please provide a summary of your Pull Request.' if github.pr_body.length < 10
+
+warn 'This is a big Pull Request.' if git.lines_of_code > 400
+
+# Require a CHANGELOG entry for non-test changes.
+if !git.modified_files.include?('CHANGELOG.md') && code_changes?
```
Comment (devsibwarra) [inline Dangerfile]: Looks like this should be `changelog.md`

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  raise 'Please include a [CHANGELOG](https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md) entry.'
```
Comment (devsibwarra) [inline Dangerfile]: `line-cookbook` => `redisio`
`CHANGELOG.md` => `changelog.md`

File: Dangerfile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+def code_changes?
+  code = %w(libraries attributes recipes resources)
+  code.each do |location|
+    return true unless git.modified_files.grep(/#{location}/).empty?
+  end
+  false
+end
+
+def test_changes?
+  tests = %w(spec test .kitchen.yml .kitchen.dokken.yml)
```
Comment (devsibwarra) [inline Dangerfile]: Kitchen files aren't hidden in this repo

File: Dangerfile (hunk 1 above)
Comment (damacus) [reply Dangerfile]: We should fix that, uppercase should/is the standard

File: Dangerfile (hunk 2 above)
Comment (damacus) [inline Dangerfile]:
```suggestion
  raise 'Please include a [CHANGELOG](https://github.com/sous-chefs/redisio/blob/master/CHANGELOG.md) entry.'
```

File: Dangerfile (hunk 1 above)
Comment (devsibwarra) [reply Dangerfile]: @Xorima You resolved these comments, but I don't see the commit that renamed the changelog file.  Just want to make sure that isn't missed

**Comment by Xorima** [comment]
It's green :D lets get this in @damacus

File: providers/configure.rb [hunk 4]
```diff
@@ -99,22 +99,18 @@ def configure
       rdb_file = current['dbfilename'] || "#{current['datadir']}/dump-#{server_name}.rdb"
 
       # Create the owner of the redis data directory
+      Chef::Log.warn("user: #{current['user']}")
+      Chef::Log.warn("systemuser: #{current['systemuser']}")
+      Chef::Log.warn("shell: #{current['shell']}")
```
Comment (damacus) [inline providers/configure.rb]: Logging?

File: changelog.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
+
 # redisio cookbook CHANGELOG
 
+## Unreleased
```
Comment (damacus) [inline changelog.md]:
```suggestion
 ## Unreleased
 Enable testing in CircleCI
```

**Comment by damacus** [comment]
Those fixes are optional, as I'd rather get this shipped :)

**Pull request merged**